core/src/builtin/arith.rs

```diff
@@ -50,5 +58,9 @@ mod test {
         let body = module.borrow().get_body().clone();
         let op = body.first().unwrap();
         assert_eq!((*op.borrow()).type_id(), TypeId::of::<ConstOp>());
+        let v1: AnyValue = From::<ConstOp>::from((constant.borrow()).clone());
```

Comment:
This doesn't look like a clean interface. Why not add `fn get_return_value(&self) -> Value</*ReturnType*/>` for types that have a return type? Also we may want to have `fn get_return_type(&self) -> Option<Type>` and `fn get_return_value(&self) -> AnyValue` for `trait Op`but that's probably another PR.

Reply:
Created new issue for this one: https://github.com/perf-toolbox/tir/issues/45